examples/aei_from_events/aei_from_events.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+
+all_cat = ['festivals', 'school-holidays', 'airport-delays', 'sports',
+           'performing-arts', 'concerts', 'public-holidays', 'community',
+           'observances', 'severe-weather', 'expos', 'conferences',
+           'health-warnings', 'academic', 'disasters', 'politics', 'terror']
+
+nonattended = ['public-holidays', 'observances', 'school-holidays', 'academic']
+
+attended = ['festivals', 'sports', 'performing-arts',
+            'concerts', 'community', 'expos', 'conferences']
+
+unscheduled = ['airport-delays', 'health-warnings', 'disasters', 'politics', 'terror', 'severe-weather']
+
+from datetime import datetime, timedelta
+
+def gen_cal(st, ed):
+    clean_start = st.replace('T', ' ').replace('Z', '')
+    start_dt = datetime.strptime(clean_start, '%Y-%m-%d %H:%M:%S')
+
+    clean_end = ed.replace('T', ' ').replace('Z', '')
+    end_dt = datetime.strptime(clean_end, '%Y-%m-%d %H:%M:%S')
+
+    date_diff = end_dt - start_dt
+    date_range = []
+    for d in range(0, date_diff.days + 1):
+        res = start_dt + timedelta(days=d)
+        date_range.append(str(res).split(' ')[0])
+    return date_range
+
+def gen_aei_from_events(cal, events_pd):
+    all_pd = []
+    for d in cal:
+        d_formatted = d + 'T00:00:00Z'
+        qualified_events = events_pd[(events_pd['start'] <= d_formatted) & (events_pd['end'] >= d_formatted)]
+        qe_agg = qualified_events.groupby('category')['impact'].sum().reset_index()
+        qe_pivot = pd.pivot_table(qe_agg, columns = 'category', values = 'impact', fill_value=0)
+        d_col = pd.DataFrame([d], columns = ['date'])
+        aei_temp = pd.concat([d_col, qe_pivot.reset_index(drop=True)], axis = 1)
+        all_pd.append(aei_temp)
+    return all_pd
+
+def attended_aei(df, event_groupings = attended):
```

Comment:
I see these functions but as a reader I am unsure on how to use it. So a few things could fix that:
* Add docstring to help explain what the func does
* Near the end of the file show how it all fits together by executing these functions - perhaps with some comments too
* You can also split the examples into 2-3 python files with clearer names